packages/ember-runtime/lib/mixins/promise_proxy.js

```diff
@@ -20,17 +21,21 @@ function tap(proxy, promise) {
 
   return promise.then(value => {
     if (!proxy.isDestroyed && !proxy.isDestroying) {
-      setProperties(proxy, {
-        content: value,
-        isFulfilled: true
-      });
+      run(() => {
```

Comment:
A run here, will change ordering and cause nested renders. I don't believe this will work